apps/web/components/eventtype/EventAvailabilityTab.tsx

```diff
@@ -87,39 +88,43 @@ const EventTypeScheduleDetails = memo(
-                      !isAvailable ? "text-subtle line-through" : "text-default"
-                    )}>
-                    {day}
-                  </span>
-                  {isPending ? (
-                    <SkeletonText className="block h-5 w-60" />
-                  ) : isAvailable ? (
-                    <div className="space-y-3 text-right">
-                      {filterDays(index).map((dayRange, i) => (
-                        <div key={i} className="text-default flex items-center leading-4">
-                          <span className="w-16 sm:w-28 sm:text-left">
-                            {format(dayRange.startTime, timeFormat === 12)}
-                          </span>
-                          <span className="ms-4">-</span>
-                          <div className="ml-6 sm:w-28">{format(dayRange.endTime, timeFormat === 12)}</div>
-                        </div>
-                      ))}
-                    </div>
-                  ) : (
-                    <span className="text-subtle ml-6 sm:ml-0">{t("unavailable")}</span>
-                  )}
-                </li>
-              );
-            })}
-          </ol>
+          {!schedule?.timeBlocks?.length ? (
+            <ol className="table border-collapse text-sm">
+              {weekdayNames(i18n.language, weekStart, "long").map((day, index) => {
+                const isAvailable = !!filterDays(index).length;
+                return (
+                  <li key={day} className="my-6 flex border-transparent last:mb-2">
+                    <span
+                      className={classNames(
+                        "w-20 font-medium sm:w-32 ",
+                        !isAvailable ? "text-subtle line-through" : "text-default"
+                      )}>
+                      {day}
+                    </span>
+                    {isPending ? (
+                      <SkeletonText className="block h-5 w-60" />
+                    ) : isAvailable ? (
+                      <div className="space-y-3 text-right">
+                        {filterDays(index).map((dayRange, i) => (
+                          <div key={i} className="text-default flex items-center leading-4">
+                            <span className="w-16 sm:w-28 sm:text-left">
+                              {format(dayRange.startTime, timeFormat === 12)}
+                            </span>
+                            <span className="ms-4">-</span>
+                            <div className="ml-6 sm:w-28">{format(dayRange.endTime, timeFormat === 12)}</div>
+                          </div>
+                        ))}
+                      </div>
+                    ) : (
+                      <span className="text-subtle ml-6 sm:ml-0">{t("unavailable")}</span>
+                    )}
+                  </li>
+                );
+              })}
+            </ol>
+          ) : (
+            schedule && <EventTypeTimeBlocks timeBlocks={schedule.timeBlocks} />
+          )}
```

Comment:
It easier to understand if in a conditional statement, we have the shorter branch written earlier
```suggestion
         {schedule && schedule?.timeBlocks?.length ? (
         <EventTypeTimeBlocks timeBlocks={schedule.timeBlocks} />)
```